db/migrate/20190925152602_create_active_admin_comments.rb

```diff
@@ -0,0 +1,16 @@
+class CreateActiveAdminComments < ActiveRecord::Migration[5.2]
+  def self.up
+    create_table :active_admin_comments do |t|
```

Comment:
Are these necessary?

Reply:
These where added by the gem on install, so I just left those, should I delete them @gonzaM ?

Reply:
I see. If you want, keep it, so you don't need to update the database, but you will no be using it.